backend/CertificateManagerAPIs/Data/CertificatedbContext.cs

```diff
@@ -33,19 +33,21 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
```

Comment:
protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
        => optionsBuilder.UseSqlServer("Name=CertificateDbConnection");

have to be deleted